.github/workflows/che.yaml

```diff
@@ -1,3 +1,19 @@
+#
+#
+# Copyright Red Hat
+#
```

Comment:
I wonder if we need to really add copyright to workflow scripts.. Do folks usually do this?

Reply:
For our repos, this can be a bit hit or miss, for this repo none of them, for registry-support there's https://github.com/devfile/registry-support/blob/a95cc889ecaf131b2ce76535ff9d9179607240be/.github/workflows/ci.yaml.

As for the open source standard for this, I'm not sure. I will investigate what other projects are doing and can revert these if its not something we want to do.

Reply:
These do not have license headers under the workflows:
- [gin-gonic/gin](https://github.com/gin-gonic/gin/tree/master/.github/workflows) (MIT)
- [kubernetes/minikube](https://github.com/kubernetes/minikube/tree/master/.github/workflows) (Apache)
- [microsoft/vscode](https://github.com/microsoft/vscode/tree/main/.github/workflows) (MIT)
- [containers/podman](https://github.com/containers/podman/tree/main/.github/workflows) (Apache)
- [redhat-developer/odo](https://github.com/redhat-developer/odo/tree/main/.github/workflows) (Apache)

These do have license headers under the workflows:
- [containers/podman-desktop](https://github.com/containers/podman-desktop/tree/main/.github/workflows) (Apache)
- [eclipse/che](https://github.com/eclipse/che/tree/main/.github/workflows) (EPL)

Looks like most don't do it, including odo.

Reply:
After reviewing Red Hat legal's recommendation for license header inclusion, I feel we should keep the license headers here.